test/BEXIS2_MetaData_Schema/create.js

```diff
@@ -0,0 +1,166 @@
+import Browser from '../../util/Browser';
```

Comment:
Please also revise all the comments. Sometimes still copy & paste from "data types" ;-). Sometimes it's missing e.g. Line 24/25.

Reply:
Comments have been modified